ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2135,5 +2135,81 @@
         "name": "type-java.svg",
         "mediaType": "image/svg+xml"
     }
-  }
+  },
+  {
+  "name": "CentOS WildFly Swarm",
+  "id": "WildFly Swarm",
+  "creator": "Dharmit Shah",
+  "description": "Eclipse WildFly Swarm Stack on CentOS.",
+  "scope": "general",
+  "source": {
+    "type": "image",
+    "origin": "registry.centos.org/che-stacks/wildfly-swarm"
+  },
+  "tags": [
+    "Java",
+    "JDK",
+    "Maven",
+    "WildFly Swarm",
+    "CentOS",
+    "Git"
+  ],
+  "workspaceConfig": {
+    "name": "default",
+    "environments": {
+      "default": {
+        "recipe": {
+          "location": "registry.centos.org/che-stacks/wildfly-swarm",
+          "type": "dockerimage"
+        },
+        "machines": {
+          "dev-machine": {
+            "attributes": {
+              "memoryLimitBytes": "2147483648"
+            },
+            "agents": [
+              "org.eclipse.che.terminal",
+              "org.eclipse.che.ws-agent",
+              "org.eclipse.che.ssh"
+            ],
+            "servers": {}
+          }
+        }
+      }
+    },
+    "commands": [
+      {
+        "name": "build",
+        "type": "mvn",
+        "attributes": {},
+        "commandLine": "scl enable rh-maven33 'mvn clean install -f ${current.project.path}'"
+      },
+      {
+        "name": "run",
+        "type": "custom",
+        "attributes": {
+          "previewUrl": "http://${server.port.8080}"
+        },
+        "commandLine": "cd ${current.project.path} && scl enable rh-maven33 'java -jar target/*-swarm.jar'"
+      }
+    ],
+    "projects": [],
+    "defaultEnv": "default",
+    "links": []
+  },
+  "components": [
+    {
+      "name": "CentOS",
+      "version": "---"
+    },
+    {
+      "name": "JDK",
+      "version": "1.8.0_45"
+    },
+    {
+      "name": "Maven",
+      "version": "3.2.2"
```

Comment:
@kenfinnigan Here's the output from inside the workspace:

```bash
$ mvn -v                                                                                                                                                                                    
Apache Maven 3.3.9 (Red Hat 3.3.9-2.8)                                                                                                                                                                           
Maven home: /opt/rh/rh-maven33/root/usr/share/maven                                                                                                                                                              
...
```
I'll modify the `stacks.json` file. Thanks for the pointer.